sdk/monitor/azure-monitor-opentelemetry-exporter/azure/monitor/opentelemetry/exporter/quickpulse/__init__.py

```diff
@@ -0,0 +1,11 @@
+# -------------------------------------------------------------------------
```

Comment:
Thought we may eventually open it to customers, wondering if we should privatize this module for now.